packages/client/Makefile

```diff
@@ -0,0 +1,9 @@
+# DU client requires the types generated from DU contracts
+unit-test: ../contracts/typechain
+	npx ganache -p 3456 -m testrpc &
```

Comment:
If `npx` is used, what version of ganache is started with what version of Node?

Reply:
whatever is installed through `npm` (listed in package.json)